Tools/CISupport/build-webkit-org/factories.py

```diff
@@ -100,7 +100,7 @@ def __init__(self, platform, configuration, architectures, additionalArguments=N
         if platform == 'wincairo':
             self.addStep(InstallWinCairoDependencies())
 
-        if platform.startswith('mac') or platform.startswith('ios-simulator'):
+        if platform.startswith('mac') or platform.startswith('ios-simulator') or platform.startswith('visionos-simulator'):
```

Comment:
can shorten these ones to something like:

`if platform.startswith(('mac', 'ios-simulator', 'visionos-simulator')):`